.github/workflows/remove-template.yml

```diff
@@ -67,7 +67,7 @@ jobs:
     outputs:
       error: ${{ steps.run-remove-from-registry.outputs.error }}
     needs: [parse-issue-form-body, validate-requestor, validate-admin]
-    if: ${{ always() && (needs.validate-requestor.outputs.error == '' || needs.validate-admin.outputs.is-admin == 'active') }}
+    if: ${{ always() && contains(join(needs.parse-issue-form-body.result, ','), 'success') && (needs.validate-requestor.outputs.error == '' || needs.validate-admin.outputs.is-admin == 'active') }}
```

Comment:
@AmyJZhao , you don't need to join `needs.parse-issue-form-body.result` as it is always `string`, you can use
```
needs.parse-issue-form-body.result == 'success'
```

Reply:
Done